sdk/translation/ai-translation-text-rest/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.0-beta.2 (Unreleased)
 
 ### Features Added
```

Comment:
Do you think it would make sense that this is adding support for AAD authentication for global endpoints?